compiler/rustc_passes/src/dead.rs

```diff
@@ -702,6 +765,21 @@ impl<'tcx> DeadVisitor<'tcx> {
                         are = pluralize!("is", span_len),
                     ));
 
+                    if is_positional {
+                        err.multipart_suggestion(
```

Comment:
The suggestion seems a bit convoluted, and may be more confusing that helping.  Since we don't give a suggestion in the non-positional case, do we need a suggestion?